mirrord/cli/src/operator.rs

```diff
@@ -71,7 +71,8 @@ async fn operator_setup(
 
 async fn get_status_api(config: Option<String>) -> Result<Api<MirrordOperatorCrd>> {
     let kube_api = if let Some(config_path) = config {
-        let config = LayerFileConfig::from_path(config_path)?.generate_config()?;
+        let mut warnings = Vec::new();
+        let config = LayerFileConfig::from_path(config_path)?.generate_config(&mut warnings)?;
```

Comment:
Maybe add add print for warnings if they happen?

Reply:
don't think we need it in this context